mongo/data_store.go

```diff
@@ -33,6 +33,36 @@ type JobStore struct {
 	cfg             *config.Config
 }
 
+// Init creates a new mgo.Session with a strong consistency and a write mode of "majority".
+func (m *JobStore) Init(ctx context.Context, cfg *config.Config) (err error) {
```

Comment:
are we expected to do some sort of testing for this?

Reply:
not as far as I know